Source/BusinessModels/BusinessModels/BusinessModels.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Rename the csproj file to Blackjack.BusinessModels for consistency with how we name our other csproj files